appinventor/appengine/src/com/google/appinventor/client/style/neo/neo.css

```diff
@@ -606,10 +606,12 @@ they get cut off in Windows.
 
   .ode-Designer-LeftColumn {
     min-width: 265px;
+    flex-shrink: 0;
```

Comment:
You have correctly pointed this PR to ucr, but you branched from master. Therefore the diff has changes from master that are not currently in ucr. You need to also branch from ucr.